v_5_2_0/files/conf/hardening.conf

```diff
@@ -0,0 +1,24 @@
+fs.inotify.max_user_watches = 16384
+# Default is 128, doubling for nodes with many pods
+# See https://github.com/giantswarm/giantswarm/issues/7711
+fs.inotify.max_user_instances = 8192
+kernel.kptr_restrict = 2
+kernel.sysrq = 0
+net.ipv4.conf.all.log_martians = 1
+net.ipv4.conf.all.send_redirects = 0
+net.ipv4.conf.default.accept_redirects = 0
+net.ipv4.conf.default.log_martians = 1
+net.ipv4.tcp_timestamps = 0
+net.ipv6.conf.all.accept_redirects = 0
+net.ipv6.conf.default.accept_redirects = 0
+# Increased mmapfs because some applications, like ES, need higher limit to store data properly
+vm.max_map_count = 262144
+# Ingress controller performance improvements
+# See https://github.com/kubernetes/ingress-nginx/issues/1939
+net.core.somaxconn=32768
+net.ipv4.ip_local_port_range=1024 65535
+# Reserved to avoid conflicts with kube-apiserver, which allocates within this range
```

Comment:
This is the only substantive change for this PR

Reply:
so the way how we do things is that adding a new version is always separate PR and then you create a new PR where you add changes. The reason for this is that in this PR its completely impossible to find any changes and when someone is trying to figure out where the change happened or what went wrong this change is un traceable.

Reply:
My bad, opened https://github.com/giantswarm/k8scloudconfig/pull/639 to add the new version